pkg/gadgets/trace/open/types/types.go

```diff
@@ -15,20 +15,25 @@
 package types
 
 import (
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	MountNsID uint64 `json:"mountnsid,omitempty"`
-	Pid       uint32 `json:"pid,omitempty"`
-	UID       uint32 `json:"uid,omitempty"`
-	Comm      string `json:"pcomm,omitempty"`
-	Fd        int    `json:"fd,omitempty"`
-	Ret       int    `json:"ret,omitempty"`
-	Err       int    `json:"err,omitempty"`
-	Path      string `json:"path,omitempty"`
+	MountNsID uint64 `json:"mountnsid,omitempty" column:"mntns,width:12,hide"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,minWidth:7,align:right"`
+	UID       uint32 `json:"uid,omitempty" column:"uid,minWidth:10,align:right,hide"`
+	Comm      string `json:"pcomm,omitempty" column:"comm,maxWidth:16"`
+	Fd        int    `json:"fd,omitempty" column:"fd,width:3,align:right,fixed"`
+	Ret       int    `json:"ret,omitempty" column:"ret,width:3,align:right,fixed,hide"`
+	Err       int    `json:"err,omitempty" column:"err,width:3,align:right,fixed"`
+	Path      string `json:"path,omitempty" column:"path,minWidth:50"`
```

Comment:
This one is difficult to choose. However I think 50 as minimum is too much. Maybe 32? Let's see others opinion on this.

Reply:
Yeah, maybe 24 or, at most, 32. We should define a value for the paths, what other gadgets have this column? Mount and fsslower?

Reply:
It differs.
For example fsslower has a `maxWidth` of `32`. That makes sense for that specific gadget since the eBPF program only captures `32` characters.
With this gadget the filepath can be up to `255` characters.

It is truly difficult to choose every columnwidth perfect for every gadget. Maybe it can't be unified as we had thought it could be.

Reason why i had chosen 50 here was just by trying different widths and running the gadget. When i felt that the gadget displayed enough of the filepath i stopped increasing the value